Parser/pegen.c

```diff
@@ -283,6 +283,8 @@ static void
 raise_tokenizer_init_error(PyObject *filename)
 {
     if (!(PyErr_ExceptionMatches(PyExc_LookupError)
+          || PyErr_ExceptionMatches(PyExc_SyntaxError)
+          || PyErr_ExceptionMatches(PyExc_UnicodeDecodeError)
```

Comment:
This isn't needed, since it's checked below.

Reply:
Whoops, bad copy paste :P